docs/cloud.md

```diff
@@ -0,0 +1,59 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in some cloud managed clusters. This document describes
+ steps to create EKS using terraform.
+
+## Common Pre-requisites
```

Comment:
s/Pre-requisites/Prerequisites

I believe it is a single word

Reply:
done